src/Ui.GraphiQL/Internal/graphiql.cshtml

```diff
@@ -56,17 +56,6 @@
     integrity="sha384-yz3/sqpuplkA7msMo0FE4ekg0xdwdvZ8JX9MVZREsxipqjU4h8IRfmAMRcb1QpUy"
     crossorigin="anonymous"
   />
-  <script
-    src="https://unpkg.com/graphiql-with-extensions@0.14.3/graphiqlWithExtensions.min.js"
-    integrity="sha384-TqI6gT2PjmSrnEOTvGHLad1U4Vm5VoyzMmcKK0C/PLCWTnwPyXhCJY6NYhC/tp19"
-    crossorigin="anonymous"
-  ></script>
-  <link
-    rel="stylesheet"
-    href="https://unpkg.com/graphiql-with-extensions@0.14.3/graphiqlWithExtensions.css"
-    integrity="sha384-GBqwox+q8UtVEyBLBKloN5QDlBDsQnuoSUfMeJH1ZtDiCrrk103D7Bg/WjIvl4ya"
-    crossorigin="anonymous"
-  />
```

Comment:
`graphiql-with-extensions` was last released 5 years ago, and its `package.json` indicates that it was designed for version 0.14.2 of GraphiQL.  It does not appear to be compatible with GraphiQL 3.x and removed it.  GraphiQL does have a docs explorer built-in, although it does not have the checkbox functionality that the `graphiql-explorer` extension did.

See:
- https://github.com/onegraph/graphiql-with-extensions
- https://github.com/OneGraph/graphiql-explorer